src/core/message-parser.ts

```diff
@@ -10,48 +11,119 @@ interface MessagePayload {
   clientId?: string;
   timestamp: number;
   extras?: {
-    timeserial?: string;
     headers?: MessageHeaders;
   };
+
+  serial: string;
+  updatedAt?: number;
+  deletedAt?: number;
+  action: Ably.MessageAction;
+  operation?: Ably.Operation;
+}
+
+interface ChatMessageFields {
+  timeserial: string;
```

Comment:
We agreed to change this to `serial` in a recent DR - have we decided which PR is going to make that change?

Reply:
Considering it's used pretty extensively, perhaps we can separate this into a follow up PR

Reply:
We'll do a PR just for that.